src/modules/Api.ts

```diff
@@ -230,18 +254,71 @@ export class Api {
     };
   }
 
+  /**
+   * Creates, but does not place a signed order
+   */
+  public async createStopLimitOrder({
+    makerAccountOwner,
+    makerMarket,
+    takerMarket,
+    makerAmount,
+    takerAmount,
+    makerAccountNumber = new BigNumber(0),
+    expiration = new BigNumber(FOUR_WEEKS_IN_SECONDS),
+    decreaseOnly,
+    triggerPrice,
+  }: {
+    makerAccountOwner: address,
+    makerAccountNumber: Integer | string,
+    makerMarket: Integer | string,
+    takerMarket: Integer | string,
+    makerAmount: Integer | string,
+    takerAmount: Integer | string,
+    expiration: Integer | string,
+    decreaseOnly?: boolean,
+    triggerPrice?: Integer,
+  }): Promise<SignedStopLimitOrder> {
+    const realExpiration: BigNumber = getRealExpiration(expiration);
+    const order: StopLimitOrder = {
+      makerAccountOwner,
+      decreaseOnly,
+      makerAccountNumber: new BigNumber(makerAccountNumber),
+      makerMarket: new BigNumber(makerMarket),
+      takerMarket: new BigNumber(takerMarket),
+      makerAmount: new BigNumber(makerAmount),
+      takerAmount: new BigNumber(takerAmount),
+      expiration: realExpiration,
+      takerAccountOwner: TAKER_ACCOUNT_OWNER,
+      takerAccountNumber: TAKER_ACCOUNT_NUMBER,
+      salt: generatePseudoRandom256BitNumber(),
+      triggerPrice: triggerPrice ? new BigNumber(triggerPrice) : triggerPrice,
```

Comment:
can just do `new BigNumber(triggerPrice)` since triggerPrice must be defined by this point